logbook-servlet/pom.xml

```diff
@@ -108,4 +108,18 @@
             <artifactId>json-path-assert</artifactId>
         </dependency>
     </dependencies>
+
+    <build>
+        <plugins>
+            <plugin>
+                <groupId>org.jacoco</groupId>
+                <artifactId>jacoco-maven-plugin</artifactId>
+                <configuration>
+                    <excludes>
+                        <exclude>org/zalando/logbook/servlet/LogbookAsyncListener.class</exclude>
```

Comment:
Why the exclude?

Reply:
@whiskeysierra thank you for the hint, I've added a simple test for LogbookAsyncListener.